src/components/navbar/HomeEmptyNav.js

```diff
@@ -1,11 +1,30 @@
 import React, { PropTypes, Component } from 'react';
 import { connect } from 'react-redux';
 import { changeStep } from '../../actions/navbarActions';
+import { getTopics } from '../../actions/topicActions';
 import smilies from '../../assets/smilies.svg';
 import UserNavBar from './UserNavBar';
 
 class HomeEmptyNav extends Component {
 
+  componentWillMount() {
+    this.props.getTopics();
+  }
+
+  renderTopics(topics) {
```

Comment:
Also move the `map` function inside this component, since you can't create a new component starting with the `map` declaration, so the new component should be:
```javascript
<li key={topic.topicId}>
  <div className="images-list">
    <div><img src={topic.icon} /></div>
    <div className="text-container"> <p>{topic.label}</p> </div>
  </div>
</li>
```